impl/picocsv/src/test/java/picocsv/PicoCsvTest.java

```diff
@@ -19,42 +21,44 @@ protected List<List<String>> parseCsvRecords(final TestSpecSettings settings, fi
         throws IOException {
         final List<List<String>> ret = new ArrayList<>();
 
-        final Csv.Reader csv = configure(input);
+        final Csv.Reader csv = configure(settings, input);
         while (csv.readLine()) {
             if (csv.isComment() && settings.commentMode() == specreader.spec.TestSpecCommentMode.SKIP) {
                 // emulating the behavior of other libraries
                 continue;
             }
 
+            if (!csv.readField()) {
+                // emulating the behavior of other libraries
+                if (settings.skipEmptyLines()) {
+                    continue;
+                }
+                ret.add(Collections.emptyList());
+                continue;
+            }
+
             final List<String> row = new ArrayList<>();
-            while (csv.readField()) {
+            do {
                 row.add(csv.toString());
-            }
+            } while (csv.readField());
 
             ret.add(row);
         }
 
         return ret;
     }
 
-    private static Csv.Reader configure(final String input) throws IOException {
+    private static Csv.Reader configure(final TestSpecSettings settings, final String input) throws IOException {
         final Csv.Format format = Csv.Format.RFC4180.toBuilder()
-            .acceptMissingField(false)
+            .acceptMissingField(settings.skipEmptyLines())
+            .comment(settings.commentMode() == TestSpecCommentMode.NONE ? '\0' : '#')
```

Comment:
Warning: This will lead to issues in the future, as binary data is also permitted in CSV according to RFC-4180-bis, section 4.2.1.